sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/KafkaBundle.java

```diff
@@ -443,9 +452,10 @@ private void setupManagedThreadManager(Environment environment) {
 
   private void stopProducers() {
     messageProducers.forEach(
-        p -> {
+        (k, v) -> {
           try {
-            p.close();
+            v.close();
+            micrometerProducerListener.producerRemoved(k);
```

Comment:
Not sure, how important this is … but if it is really important, this call might be moved in a finally block.

Reply:
moved to finally